native/storybook/storybook.js

```diff
@@ -15,23 +15,41 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 import React from 'react';
 import { I18nManager, AppRegistry, StyleSheet, View } from 'react-native';
 import RNRestart from 'react-native-restart';
+import addon from '@storybook/addons';
 import {
   getStorybookUI,
   configure,
   addDecorator,
 } from '@storybook/react-native';
 
-import BpkButton from '../packages/react-native-bpk-component-button';
 import {
   spacingBase,
   spacingLg,
   spacingXxl,
 } from './../../packages/bpk-tokens/tokens/base.react.native';
 
+const toggleRTL = rtlEnabled => {
+  I18nManager.forceRTL(!rtlEnabled);
+  RNRestart.Restart();
+};
+
+const onChannelAvailable = (...fns) => {
+  const interval = setInterval(() => {
+    try {
+      const channel = addon.getChannel();
+      clearInterval(interval);
+      fns.map(fn => fn(channel));
+    } catch (exe) {
+      console.log(exe);
+    }
+  }, 500);
```

Comment:
The logic here is a great workaround, but let's add a quick comment to explain why it's necessary, as it's not immediately obvious why we're doing this.

Reply:
`addon.getChannel()` will return the channel responsible of sending events back and forth to RN, the channel is create at render time so not availalbe when this file first execute, once the channel is create, it'll be avaialbe and ready to use

Reply:
I meant a comment in the code haha 😂